dev/io.openliberty.org.apache.myfaces.4.1_fat/test-applications/Flow_Spec1342_Spec1739.war/resources/simple/simple-flow.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ * Copyright (c) 2015, 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ -->
+  <!-- Copied from JSF 2.2 FAT adn Modified-->
+<faces-config version="2.2" xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-facesconfig_2_2.xsd">
```

Comment:
new version of the schema? If not perhaps open an issue so we refresh these once available?

Reply:
Schemas are not yet released (see https://jakarta.ee/xml/ns/jakartaee/) , so I'll take to update them. 
Created: https://github.com/OpenLiberty/open-liberty/issues/27801